gpii/node_modules/lifecycleManager/src/LifecycleManagerUtils.js

```diff
@@ -0,0 +1,148 @@
+/*!
+ * Lifecycle Manager Utilities
+ *
+ * Copyright 2019 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013)
+ * under grant agreement no. 289016.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = fluid || require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.lifecycleManager");
+
+(function () {
```

Comment:
This was an ok practice from the previous decade but unnecessary I think since we don't close over anything or allocate any globals